### PR TITLE
Fixed up sign on Driver Torque, added unit

### PR DESCRIPTION
**Pull request by avolmensky**
Fixing DriverTorque sign as lane change is not working properly

### CODE REVIEW

File: nissan_2017.dbc [hunk 1]
```diff
@@ -86,7 +86,7 @@ BO_ 389 Steering: 8 XXX
  SG_ LKAS_Torque : 47|8@0+ (1,0) [0|255] "" XXX
  SG_ CRC : 63|8@0+ (1,0) [0|127] "" XXX
  SG_ Angle : 23|18@0+ (-0.01,1310) [0|262143] "" XXX
- SG_ DriverTorque : 7|12@0+ (1,-2047) [0|4095] "" XXX
+ SG_ DriverTorque : 7|12@0+ (-1,2047.5) [0|4095] "Ncm" XXX
```
Comment (rbiasini) [inline nissan_2017.dbc]: what's `Ncm`? :D

Comment (rbiasini) [reply nissan_2017.dbc]: also, the offset should be an integer. 2047 or 2048?

Comment (avolmensky) [reply nissan_2017.dbc]: Ncm = Nm * 100. Should I scale this to Nm?

Comment (rbiasini) [reply nissan_2017.dbc]: yeah, set the factor to 0.01

**Pull request merged**